CHANGELOG.md

```diff
@@ -1,4 +1,48 @@
 # Changelog
+## Friday 12th August 2022
+## Docs updates 
+We launched a [Discussion](https://docs.api.audio/discuss) page so you can ask us your questions. And a [FAQs](https://docs.api.audio/discuss?isFAQ=true) page. Check this out!
+
+## Dictionary (Lexi)
+We updated our Dictionary feature - to make it even more user friendly. 
+We call our Dictionary feature - Lexi.
```

Comment:
Let's remove this part and keep the feature name consistently Pronunciation Dictionary

Reply:
do you mean to deprecate the wording completely? They will all be called Text Processors in the future

Reply:
Yeah we shouldnt use LEXI at all. So until we change the name it should be Pronunciation Dictionary

Reply:
I could have changed this?. I guess you mean

apiaudio.lexi -> apiaudio.pronuncation?

Reply:
Yes that would be good if we can also change the routes to that.